tests/test_utils.py

```diff
@@ -231,4 +238,46 @@ def test_get_bib_record(mock_requests_get):
 def test_get_marc_from_bib():
     bib_record = open(str(Path(__file__).parent / "data/example_bib_record.xml"), "rb").read()
     record = open(str(Path(__file__).parent / "data/example_marc.xml"), "rb").read()
-    assert etree.tostring(get_marc_from_bib(bib_record)) ==  record
\ No newline at end of file
+    assert etree.tostring(get_marc_from_bib(bib_record)) ==  record
+
+def test_get_digitized_bags(mock_celery_backend):
+    results = [{'bag':'shareok/bagname_123456789',
+               'locations.s3.exists': True
+              },
+               {'bag':'shareok/digitized',
+               'locations.s3.exists': True,
+               }
+              ]
+    mock_celery_backend.database.client.catalog.digital_objects.find.return_value = results
+    assert get_digitized_bags('123456789') == ['bagname_123456789', 'digitized']
+    
+def test_update_ingest_status(mock_celery_backend):
+    mock_celery_backend.database.client.catalog.digital_objects.find_one.return_value = {'bag':'shareok/bagname', '_id': 'aaaaaaaaaaaaaaaaaaaaaaaa'}
+    mock_celery_backend.database.client.catalog.digital_objects.update.return_value = {'ok': 1}
```

Comment:
This should return {'nModified': 1} to match the change in the update_ingested_status function in that utils.py file.